extensions/mssql/src/objectManagement/objectManagementService.ts

```diff
@@ -432,6 +435,32 @@ export class TestObjectManagementService implements IObjectManagementService {
 		};
 	}
 
+	private getServerPropertiesView(isNewObject: boolean, name: string): ServerPropertiesViewInfo {
+		return isNewObject ? <ServerPropertiesViewInfo>{
```

Comment:
I don't think we have any plans to support "new server" from here (we have stuff like deployment for that) so can probably remove that logic.

@erinstellato-ms Any thoughts?

Reply:
@Charles-Gagnon Agree, we don't need "new server" (you're correct in that would only happen via deployment, or external to ADS).